docker-nginx/Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM nginx:stable
 
 COPY pages /var/www/html/pages/
 COPY templates/ /etc/nginx/templates/
+COPY dhparams/ /etc/nginx/dhparams/
 COPY options-ssl-nginx.conf /etc/nginx/options-ssl-nginx.conf
 COPY 99-autoreload.sh /docker-entrypoint.d/99-autoreload.sh
-CMD chmod +x /docker-entrypoint.d/99-autoreload.sh
+RUN chmod +x /docker-entrypoint.d/99-autoreload.sh
```

Comment:
This was actually the missing bit that blocked the base PR. Thanks for spotting and fixing!